src/core/widget-config.ts

```diff
@@ -1,5 +1,7 @@
 import { ITemplateMeta } from "./template";
 
+const DX_PREFIX: string = "_dx";
```

Comment:
It breaks naming conventions for private\public members and we're referring these special props in our code.
I suggest using smth like `dx_` instead.